### PR TITLE
Anadir ejercito

### PR DESCRIPTION
**Pull request by cahegigithub**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2053)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: espanol/ejercito/ejercito.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Y vinieron un ejército de ángeles y se pusieron a tocar el violín
```
Comment (casey-collab) [inline espanol/ejercito/ejercito.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: espanol/ejercito/ejercito.md (hunk 1 above)
Comment (cahegigithub) [inline espanol/ejercito/ejercito.md]: Once corrected this, shoud a make another pull request or how do i do it?

Comment (cahegigithub) [inline espanol/ejercito/ejercito.md]: I have made the story reacheable(i think i made it well) and push me changes into my changes branch

**Comment by cbuckey-uda** [comment]
@cahegigithub, pushing your changes made them show up in this pull request, so there's no need to make another!

@casey-collab, please take another look

File: espanol/canudos.md [hunk 2]
```diff
@@ -51,3 +51,4 @@ el Consejero:
 [Empieza a gritar como un poseso](poseso/poseso.md)
 
 [Se arrodilla y...](rogando/rogando.md)
+[Y vino un ejercito](ejercito/ejercito.md)
```
Comment (casey-collab) [inline espanol/canudos.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

**Comment by casey-collab** [comment]
Looks great! :penguin:
